.github/workflows/backend-tests.yml

```diff
@@ -124,13 +124,13 @@ jobs:
           fi
```

Comment:
The changes to this file are in a dedicated PR: https://github.com/dfinity/oisy-wallet/pull/6245